Kitodo/src/main/webapp/WEB-INF/templates/includes/processFromTemplate/dialogs/selectProject.xhtml

```diff
@@ -0,0 +1,67 @@
+<!--
+ *
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ *
+-->
+
+<ui:composition
+        xmlns="http://www.w3.org/1999/xhtml"
+        xmlns:h="http://xmlns.jcp.org/jsf/html"
+        xmlns:f="http://xmlns.jcp.org/jsf/core"
+        xmlns:ui="http://xmlns.jcp.org/jsf/facelets"
+        xmlns:p="http://primefaces.org/ui">
+
+    <p:dialog id="selectProjectDialog"
+              widgetVar="selectProjectDialog"
+              modal="true"
+              resizable="false"
+              dynamic="true"
+              appendTo="@(body)"
+              width="480px">
+        <h3>#{msgs['selectProject']}</h3>
+        <h:form id="selectProjectForm">
+            <h:panelGroup styleClass="select-note ui-messages-info" layout="block">#{msgs['multipleProjectsText']}</h:panelGroup>
+            <h:panelGroup styleClass="select-selector" layout="block">
+                <p>
+                    <p:selectOneMenu id="projectMenu"
+                                     required="#{not empty param['selectProjectForm:setProjectButton']}"
+                                     value="#{SelectProjectDialogView.selectedProjectId}">
+                        <f:selectItem itemValue="#{null}"
+                                      itemLabel="#{msgs['selectProject']}"
+                                      noSelectionOption="true"/>
+                        <f:selectItems value="#{SelectProjectDialogView.templateProjects}"
+                                       var="project"
+                                       itemLabel="#{project.title}"
+                                       itemValue="#{project.id}"/>
+                        <p:ajax update="selectProjectForm:setProjectButton"/>
+                    </p:selectOneMenu>
+                </p>
+                <p:panelGrid>
+                    <p:row>
+                        <p:commandButton id="setProjectButton"
+                                         action="#{SelectProjectDialogView.createProcessForTemplate(SelectProjectDialogView.selectedProjectId, SelectProjectDialogView.template.id)}"
```

Comment:
I think you don't need to explicitly give project id and selected template here, as it is know within the class, right?